go.mod

```diff
@@ -193,8 +233,16 @@ require (
 	modernc.org/sqlite v1.14.5 // indirect
 	modernc.org/strutil v1.1.1 // indirect
 	modernc.org/token v1.0.0 // indirect
+	sigs.k8s.io/json v0.0.0-20211020170558-c049b76a60c6 // indirect
+	sigs.k8s.io/kustomize/api v0.10.1 // indirect
+	sigs.k8s.io/kustomize/kyaml v0.13.0 // indirect
+	sigs.k8s.io/structured-merge-diff/v4 v4.2.1 // indirect
+	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
+// TODO(josedonizetti): replace once the PRs are merged on upstream trivy
```

Comment:
Looks like PRs are already in the upstream (trivy-kubernetes). Do we still need the `replace`?

Reply:
Yes, I'm waiting for a last merge -> https://github.com/aquasecurity/trivy-kubernetes/pull/12

Reply:
OK, let's wait for it then.